syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -88,16 +97,71 @@ def float_precision(self):
 
         return result
 
+    def truncate(self, precision_fractional):
+        truncation = self.base ** precision_fractional
+        self.child /= truncation
+        return self
+
     @hook
     def add(self, _self, *args, **kwargs):
         """Add two fixed precision tensors together.
-
-        TODO: fix!
         """
         response = getattr(_self, "add")(*args, **kwargs)
 
         return response
 
+    __add__ = add
```

Comment:
Does this work to replace it?

Reply:
To replace what?

Reply:
the `__add__` method

Reply:
Yes ! That's actually a thing Pytorch does :)
See https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L360